man.py

```diff
@@ -6,9 +6,12 @@
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "volunteer_planner.settings.local")
     os.environ.setdefault("DATABASE_PW", "")
     os.environ.setdefault("DB_USER", "")
+    os.environ.setdefault("SECRET_KEY", "1")
+    os.environ.setdefault("LANGUAGE", "de")
```

Comment:
As far as I understand, you don't have to set default language here. Django will take from settings file.